package.json

```diff
@@ -28,42 +28,46 @@
   "sideEffects": false,
   "exports": {
     ".": {
-      "types": "./index.d.ts",
-      "default": "./index.js"
+      "types": "./dist/index.d.ts",
+      "default": "./dist/index.js"
     },
-    "./*": {
-      "types": "./*/index.d.ts",
-      "default": "./*/index.js"
+    "./plugins/*": {
+      "types": "./dist/plugins/*/index.d.ts",
+      "default": "./dist/plugins/*/index.js"
+    },
+    "./locales/*": {
+      "types": "./dist/locales/*.d.ts",
+      "default": "./dist/locales/*.js"
     }
   },
-  "main": "index.js",
-  "module": "index.js",
-  "types": "index.d.ts",
+  "main": "./dist/index.js",
+  "module": "./dist/index.js",
+  "types": "./dist/index.d.ts",
   "typesVersions": {},
   "files": [
-    "*"
+    "dist",
+    "README.md"
   ],
   "scripts": {
     "lint": "pnpm exec biome check .",
     "lint:fix": "pnpm exec biome check --write .",
-    "build": "tsup --dts",
+    "build": "tsdown",
     "build:ghpages": "node ./scripts/gh-pages/create-gh-pages.js",
     "type-check": "tsc --noEmit --skipLibCheck",
     "test": "vitest",
     "test:ui": "vitest --ui --api 9527",
     "test:run": "vitest --run",
-    "test:run:tz1": "TZ=Pacific/Auckland vitest --run",
-    "test:run:tz2": "TZ=Asia/Kathmandu vitest --run",
+    "test:run:tz1": "cross-env TZ=Pacific/Auckland vitest --run",
```

Comment:
cross-env does not make sense in this case, as this script does not have any effect on windows: you cannot set the timezone via an environment variable in windows.
So perhaps we can go back to the version without cross-env and have 1 direct decDependeny less (i.e. less trouble with security issues in deps 😄 ).